.github/infrastructure/terraform/conformance/pubsub/aws/snssqs/snssqs.tf

```diff
@@ -0,0 +1,94 @@
+resource "aws_sns_topic" "testTopic" {
+  name = "testTopic"
```

Comment:
do we need enforce uniqueness on these identifiers? What would happen if two PRs try to run certification/conformance tests at the same time? Would those collide? If so, should we suffix those with `${github.run_id}`